test/torch/model/test_deepar_modules.py

```diff
@@ -145,3 +145,81 @@ def test_deepar_modules(
 
     assert lightning_module.training_step(batch, batch_idx=0).shape == ()
     assert lightning_module.validation_step(batch, batch_idx=0).shape == ()
+
+
+@pytest.mark.parametrize(
+    "prediction_length, context_length, lags_seq",
+    [
+        (1, 5, [1, 10, 15]),
+        (1, 5, [3, 6, 9, 10]),
+        (2, 5, [1, 2, 7]),
+        (2, 5, [2, 3, 4, 6]),
+        (4, 5, [1, 5, 9, 11]),
+        (4, 5, [7, 8, 13, 14]),
+    ],
+)
+def test_rnn_input(
+    prediction_length: int, context_length: int, lags_seq: List[int]
+):
+    num_samples = 10
+    history_length = max(lags_seq) + context_length - 1
+
+    model = DeepARModel(
+        freq="D",
+        prediction_length=prediction_length,
+        context_length=context_length,
+        lags_seq=lags_seq,
+        scaling=False,
+        num_parallel_samples=num_samples,
+    )
+
+    batch = {
+        "feat_static_cat": torch.tensor([[0]], dtype=torch.int64),
+        "feat_static_real": torch.tensor([[0.0]], dtype=torch.float32),
+        "past_time_feat": torch.arange(history_length, dtype=torch.float32)
+        .unsqueeze(0)
+        .unsqueeze(-1),
```

Comment:
I think I prefer `reshape(1, history_length, 1)` instead of `unsqueeze`. Makes it more clear what the output shapes is in my view.

Reply:
or even `.view()`